### ISSUE TITLE
gist id not available on creating new gist

### ISSUE THREAD

**Issue by shanalikhan** [octokit/octokit.js#981]
Hi, 
I recently got depreciated warning that gist `id` will be replaced by `gist_id`.
I am trying to replace in my project.

This takes `gist_id`
`this.github.gists.get({ gist_id: GIST });`

But 

`this.github.gists.create(this.GIST_JSON_EMPTY);`

always return object with **id** attribute rather than `gist_id`.
Please clarify that which attribute will be used on which action or `id` will be replace in next version also.

As when i get the GIST with `id` attribute and try to save that gist object it tells depreciation error

**Comment by gr2m** [comment]
`github.gists.create()` returns the response of the REST API endpoint, which includes an `id` attribute for the Gist ID: https://developer.github.com/v3/gists/#response-5

`github.gists.get({ gist_id: GIST })` sends a request to [`GET /gists/:gist_id`](https://developer.github.com/v3/gists/#get-a-single-gist). All the `id` properties in the routes documentation have been prefixed with the entity name, hence the property name `gist_id`.

So to summarise, `github.gists.create()` will continue to return an `id` property. But `github.gists.get({id})` is deprecated in favour of `github.gists.get({gist_id})`

Let me know if you have further questions
